src/main/java/Problem1/LevelOrderTraverse.java

```diff
@@ -1,12 +1,47 @@
 package Problem1;
 
 import java.util.ArrayList;
+import java.util.LinkedList;
 import java.util.List;
+import java.util.Queue;
 
 public class LevelOrderTraverse {
     public static List<List<Integer>> levelOrder(TreeNode<Integer> root) {
-        // homework
+
+
         List<List<Integer>> result = new ArrayList<>();
-        return result;  // place holder
+        if (root == null) {
+            return result;
+        }
+
+        Queue<TreeNode> q = new LinkedList<TreeNode>();
+        q.add(root);
+        q.add(null);
```

Comment:
why add null?

Reply:
Null is used a partition (marker) to keep track where the current level ends